docs/source/publish_and_share_kedro_viz_on_aws.md

```diff
@@ -44,55 +56,34 @@ export AWS_SECRET_ACCESS_KEY="your_secret_access_key"
 
 For more information, see the official AWS documentation about [how to work with credentials](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html).
 
-### Permissions and access control
-
-All permissions and access control are controlled by AWS. It's up to you, the user, if you want to allow anyone to see your project or limit access to certain IP addresses, users, or groups.
-
-You can control who can view your visualisation using [bucket and user policies](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-iam-policies.html) or [access control lists](https://docs.aws.amazon.com/AmazonS3/latest/userguide/acls.html). See the official AWS documentation for more information.
 
 ### Billing
 
 You pay for storing objects in your S3 buckets. The amount you pay depends on your objects’ size, how long you stored the object during the month, and the storage class.
 
 See the official [AWS documentation](https://aws.amazon.com/s3/pricing/?nc=sn&loc=4) for more information.
 
-## Step 4: Publish and share the project
-Once your Cloud storage is configured and the credentials are set, you are now ready to publish and share your Kedro-Viz project. Start Kedro-Viz by running the following command in your terminal:
+## Publish and share the project
+
+Step 9: Once your Cloud storage is configured and the credentials are set, you are now ready to publish and share your Kedro-Viz project. Start Kedro-Viz by running the following command in your terminal:
 
 ```bash
 kedro viz run
 ```
 
-Click the **Publish and share** icon in the lower-left of the application. You will see a modal dialog to select your relevant AWS Bucket Region and enter your Bucket Name.
+Step 10: Click the **Publish and share** icon in the lower-left of the application. You will see a modal dialog to select your relevant AWS Bucket Region and enter your Bucket Name.
 
 ```{note}
-From Kedro-Viz version 8.0.0, you will see a modal dialog to select your hosting platform, input your bucket name and endpoint link. The endpoint link can be found under **S3 bucket -> properties -> Static website hosting -> Bucket website endpoint**.
+You will see a modal dialog to select your hosting platform, input your bucket name and endpoint link. The endpoint link can be found under **S3 bucket -> properties -> Static website hosting -> Bucket website endpoint**.
```

Comment:
since aws sharing had a different modal before 8.0.0, it might be worth mentioning, **From Kedro-Viz version 8.0.0** here